shared_model/bindings/model_query_builder.cpp

```diff
@@ -37,15 +37,20 @@ namespace shared_model {
     }
 
     ModelQueryBuilder ModelQueryBuilder::getAccountTransactions(
-        const interface::types::AccountIdType &account_id) {
-      return ModelQueryBuilder(builder_.getAccountTransactions(account_id));
+        const interface::types::AccountIdType &account_id,
+        interface::types::TransactionsNumberType page_size,
+        const boost::optional<interface::types::HashType> &first_hash) {
```

Comment:
It is highly likely that optional is not wrapped with SWIG in an expected way. Please check java bindings and comment if it was successful or not.